src/include/btmem.h

```diff
@@ -277,14 +268,9 @@ struct __wt_multi {
     uint32_t supd_entries;
     bool supd_restore; /* Whether to restore saved update chains to this page */
 
-    /*
-     * Disk image was written: address, size and checksum. On subsequent reconciliations of this
-     * page, we avoid writing the block if it's unchanged by comparing size and checksum; the reuse
-     * flag is set when the block is unchanged and we're reusing a previous address.
-     */
+    /* Disk image was written: address and size. */
     WT_ADDR addr;
     uint32_t size;
```

Comment:
Suggestion: if we change the line
    `dsk->mem_size = multi->size;` 
to 
    `dsk->mem_size = WT_STORE_SIZE(chunk->image.size);` 
in `__rec_split_write` we no longer need this `size` field.

Reply:
Thanks. Fixed.